src/database/card.ts

```diff
@@ -0,0 +1,20 @@
+import Card from "../schemas/Card";
+import { getMongoDBClient } from "./database";
+
+const dbClient = getMongoDBClient();
+const dbName = "carddb";
+const collectionName = "Card";
+
+export async function insertCard(data: Card[]) {
+  await dbClient.connect();
```

Comment:
Connection to DB should always be open. Establish connection along with spinning up the server, and reuse DB instance on every API endpoint without reconnecting again. 
[Reference](https://www.mongodb.com/resources/languages/express-mongodb-rest-api-tutorial).

Reply:
I already reuse the connected client for the whole server, but let me check out the suggested reference.

Reply:
The same client instance inefficiently gets triggered to establish a connection again.

Reply:
I see. I forgot to delete it.